.travis.yml

```diff
@@ -55,10 +54,7 @@ env:
 jobs:
   include:
     - rvm: 2.6.5
-      script: bin/rake rubocop
-      stage: linting
-    - rvm: 2.6.5
-      script: bin/rake check_rvm_integration
+      script: bin/rake rubocop check_rvm_integration man:check
```

Comment:
Does this continue on first fail? Let's say `rubocop` will fail, is `man:check` triggered as well?

Reply:
No, it aborts.

Reply:
Then the question is if you would like to trade 1 minute for a little less comfort. Having multiple jobs in linting phase makes it easier to see problems (since you'll see which job failed really quickly without need to open the log).

Reply:
Yeah, as I said, I don't have a strong opinion either way.